src/js/components/Article.js

```diff
@@ -422,6 +424,19 @@ export default class Article extends Component {
     }
   }
 
+  _renderPagination () {
+    if (this.state.atBottom) {
+      const childCount = React.Children.count(this.props.children);
+      return (
+        <Box className="article__pagination" align="center">
+          <Box align="center">
+            <Paragraph size="large">{this.state.selectedIndex+1} of {childCount}</Paragraph>
```

Comment:
minor suggestion: can we add a space in the operator?

this.state.selectedIndex + 1